.tidyallrc

```diff
@@ -0,0 +1,3 @@
+[PerlTidy]
+select = **/*.{pl,pm,t}
```

Comment:
This ignores all perl scripts without a file extension, e.g. under `script/`.

Reply:
https://github.com/os-autoinst/openQA/pull/4485 should fix that